pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/TravelAddress"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid input"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "StatusReply": {
+        "type": "object",
+        "description": "The response object for the status endpoint.",
+        "properties": {
+          "status": {
             "type": "string",
-            "format": "date-time"
+            "example": "ok",
+            "enum": [
+              "ok",
+              "not ready",
+              "unhealthy",
+              "maintenance"
+            ],
+            "description": "The status of the node."
           },
-          "received_at": {
+          "uptime": {
             "type": "string",
-            "format": "date-time"
+            "format": "duration",
+            "example": "25h33m29.4860s",
+            "description": "The duration since the node was last started."
+          },
+          "version": {
+            "type": "string",
+            "format": "semver",
+            "example": "1.0.0",
+            "description": "The version of the node."
           }
         }
       },
-      "Pending": {
+      "APIAuthentication": {
         "type": "object",
+        "description": "The request object for the authentication endpoint.",
         "properties": {
-          "envelope_id": {
-            "type": "string"
-          },
-          "received_by": {
-            "type": "string"
+          "client_id": {
+            "type": "string",
+            "description": "The client ID for authentication."
           },
-          "received_at": {
+          "client_secret": {
+            "type": "string",
+            "description": "The client secret for authentication."
+          }
+        }
+      },
+      "LoginReply": {
+        "type": "object",
+        "description": "The response object for the authentication endpoint.",
+        "properties": {
+          "access_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The access token for the authenticated session."
           },
-          "message": {
-            "type": "string"
+          "token_type": {
+            "type": "string",
+            "description": "The type of the access token."
           },
-          "reply_not_after": {
+          "refresh_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The refresh token for obtaining a new access token."
           },
-          "reply_not_before": {
+          "expires_in": {
```

Comment:
The `LoginReply` found [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/api/v1/auth.go#L20-L23) doesn't include `expires_in`. I believe this should be removed.